src/index.ts

```diff
@@ -188,7 +188,9 @@ function _preprocess(content: string, filename: string) {
       if (FINAL_TEXT_MATCHES) {
         // console.log(FINAL_TEXT_MATCHES);
         let extractedClasses = FINAL_TEXT_MATCHES[3];
-        let INLINE_EXPRESSION = FINAL_TEXT_MATCHES[3].toString().match(/([\{].*?[\}])/gi);
+
+        // Find ternary operators and extract the classes
+        let INLINE_EXPRESSION = FINAL_TEXT_MATCHES[3].toString().match(/([\{].*[\?].*[\:].*[\}])/gi);
```

Comment:
does not work

Reply:
Are you sure?
This is what I get before my change:
![image](https://user-images.githubusercontent.com/46045829/110941765-2316bf80-8339-11eb-878b-9b8361aa83ab.png)

And after modifying the regex it works

Reply:
you need to look at following class example

`class="divide-x bg-gray-800 animate-pulse text-gray-100 test {blue ? 'test' : 'why'} language-{size} {className}"`

we need to recognize `{blue ? 'test' : 'why'}` AND `{className}` as inline-expression
only `language-{size}` should be ignored

Reply:
so regex should only ignore curly brackets if combined with another word.. if standalone it should handle as inline expression. can you adapt your regex an i will re-review